src/pages/UserPreference.tsx

```diff
@@ -0,0 +1,77 @@
+import { ReactElement } from "react";
+import "tw-elements";
+
+export type Member = {
+  id: String;
+  firstName: String;
+  lastName: String;
+  email: String;
+  activeMember: Boolean;
+  votingRights: Boolean;
+  receiveNotPresentEmail: Boolean;
+  includeInQuorum: Boolean;
+  signInBlocked: Boolean;
+};
+
+export type UserPreferenceProp = {
+  member: Member;
+};
+
+const UserPreference = ({ member }: UserPreferenceProp): ReactElement => {
+  return (
+    <div className="flex flex-col flex-1 p-4 font-sans md:p-10 gap-y-8">
+      <div className="flex flex-col font-sans font-bold">
+        <span className="text-gray-600 text-xl"> Hello,</span>
+        <span className="section-heading m-0"> NORTHEASTERN SGA</span>
+      </div>
+
+      <div className="flex flex-col px-7 font-sans gap-8">
+        <span className="font-bold text-xl">CONTACT INFO</span>
+        <div>
+          <span className="text-gray-600">Name</span> <br />
+          {member.firstName + " " + member.lastName}
+        </div>
+        <div className="flex flex-col">
+          <span className="text-gray-600">Email</span>
+          <span>{member.email}</span>
+        </div>
+
+        <hr className="border-black home-mx -ml-7 -mr-7" />
+
+        <div className="flex flex-col font-sans font-bold">
+          <span>YOUR GROUPS</span>
+          <div className="flex flex-row flex-wrap gap-6 p-4 text-sm">
```

Comment:
Super nitpick sorry lol: Maybe would add padding only for the top and bottom of these tags, since in the wireframe the "YOUR GROUPS" text is aligned with the tags?